lib/instance-validator.js

```diff
@@ -321,10 +321,7 @@ class InstanceValidator {
     let error;
 
     if (rawAttribute.allowNull === false && (value === null || value === undefined)) {
-      const validators = this.modelInstance.validators[field];
-      const errMsg = validators
-                      ? (validators.notNull || {}).msg
-                      : `${field} cannot be null`;
+      const errMsg = `${field} cannot be null`;
```

Comment:
This now doesn't use `validators.notNull.msg` anymore, whereas it did before. I would suggest
```js
const errMsg = validators && validators.notNull && validators.notNull.msg || `${field} cannot be null`
```